services/app-api/src/resolvers/healthPlanPackage/submitHealthPlanPackage.ts

```diff
@@ -367,9 +367,9 @@ export function submitHealthPlanPackageResolver(
             }
 
             // If there are rates, submit those first
-            if (contractWithHistory.revisions[0].rateRevisions.length > 0) {
+            if (contractWithHistory.draftRevision.rateRevisions.length > 0) {
```

Comment:
I think this wants to be contractWithHistory.draftRevision.draftRates ?

Reply:
NO I think I'm wrong, at this point it's been turned into our domain model which doesn't have draftRates